hydromt_fiat/api/exposure_vm.py

```diff
@@ -286,7 +289,7 @@ def set_roads_settings(
         self.exposure_roads_model = ExposureRoadsSettings(
             roads_fn="OSM",
             road_types=road_types,
-            road_damage=1,
+            road_damage= math.nan,
```

Comment:
why are you using the math.nan ? I haven't seen this before. Why not "None" or "np.nan" if you know it is a float?